great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -320,9 +321,9 @@ def get_batch_data_and_markers(
                 reader_fn: Callable = self._get_reader_fn(
                     reader=reader,
                     reader_method=reader_method,
-                    path=path,
+                    path=reader_path,
                 )
-                batch_data = reader_fn(path)
+                batch_data = reader_fn(reader_path)
```

Comment:
@lccasagrande It still says `reader_path` instead of `path` when I am looking at it.  Thanks.

Reply:
Done